Ghost filter/Ghost filter code.py

```diff
@@ -0,0 +1,34 @@
+from matplotlib import pyplot as plt
+import numpy as np
+import cv2
+
+img_path = r"path/of/the/image"  #path of the image
```

Comment:
path of image needs to be taken from the user, example:
```python
num = int(input("Enter a number: "))
```

Reply:
done!